### PR TITLE
feat(web): mobile view top jurors and some refactors

### PR DESCRIPTION
**Pull request by kemuru**
One thing to note is I created MobileCard.tsx and DesktopCard.tsx (because the components had different structures between desktop and mobile versions). Lmk what you think (we did the same with Navbar IIRC).

---

## PR-Codex overview
This PR focuses on updating the TopJurors component in the Home page. 

### Detailed summary
- Deleted JurorCard.tsx, TopJurorsHeader.tsx, and Rank.tsx.
- Added Rank.tsx, JurorTitle.tsx, Coherency.tsx, HowItWorks.tsx, Rewards.tsx, and index.tsx.
- Updated WithHelpTooltip.tsx.
- Updated JurorCard/index.tsx.
- Updated TopJurors/index.tsx.
- Updated JurorCard/Rank.tsx.
- Updated JurorCard/JurorTitle.tsx.
- Updated Header/HowItWorks.tsx.
- Updated JurorCard/HowItWorks.tsx.
- Updated Header/Rewards.tsx.
- Updated Header/Coherency.tsx.
- Updated Header/index.tsx.
- Updated JurorCard/Rewards.tsx.

> The following files were skipped due to too many changes: `web/src/pages/Home/TopJurors/JurorCard/Rewards.tsx`, `web/src/pages/Home/TopJurors/JurorCard/DesktopCard.tsx`, `web/src/pages/Home/TopJurors/JurorCard/MobileCard.tsx`, `web/src/assets/svgs/menu-icons/help.svg`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: web/src/pages/Home/TopJurors/Header/JurorTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  width: calc(40px + (220 - 40) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+  gap: 36px;
+
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/Header/JurorTitle.tsx:10]: Don't need label

File: web/src/pages/Home/TopJurors/Header/Rank.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  width: calc(16px + (24 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/Header/Rank.tsx:7]: I think label is not needed here either.

File: web/src/pages/Home/TopJurors/Header/Coherency.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  ${landscapeStyle(
+    () =>
+      css`
+        display: flex;
+
+        label {
+          font-size: 14px !important;
+          &::before {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/Header/Coherency.tsx:22]: Try adding the pseudoelement to the tooltip

File: web/src/pages/Home/TopJurors/JurorCard/Coherency.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/JurorCard/Coherency.tsx:7]: Label is not needed here

File: web/src/pages/Home/TopJurors/JurorCard/DesktopCard.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 100%;
+  background-color: ${({ theme }) => theme.whiteBackground};
+  border: 1px solid ${({ theme }) => theme.stroke};
+  border-top: none;
+  align-items: center;
+  padding: 15.55px 32px;
+
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/JurorCard/DesktopCard.tsx:22]: try removing this

File: web/src/pages/Home/TopJurors/JurorCard/HowItWorks.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import { landscapeStyle } from "styles/landscapeStyle";
+import PixelArt from "pages/Dashboard/JurorInfo/PixelArt";
+import { getUserLevelData } from "utils/userLevelCalculation";
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/JurorCard/HowItWorks.tsx:12]: Make it into it's own styled component

File: web/src/pages/Home/TopJurors/JurorCard/MobileCard.tsx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+  flex-wrap: wrap;
+  width: 100%;
+  background-color: ${({ theme }) => theme.whiteBackground};
+  padding: 16px 24px 24px 24px;
+  border 1px solid ${({ theme }) => theme.stroke};
+  border-top: none;
+  align-items: center;
+  gap: 20px;
+
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/JurorCard/MobileCard.tsx:24]: I think we can remove this style too

File: web/src/pages/Home/TopJurors/JurorCard/Rank.tsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { landscapeStyle } from "styles/landscapeStyle";
+
+const Container = styled.div`
+  label {
```
Comment (alcercu) [inline web/src/pages/Home/TopJurors/JurorCard/Rank.tsx:6]: either div or label, doesn't need both

**Review by alcercu — CHANGES_REQUESTED** [review]
Resolve rest of comments

**Review by jaybuidl — APPROVED** [review]

**Pull request merged**